test-plans/Makefile

```diff
@@ -0,0 +1,10 @@
+all: ping-image.tar
+
+ping-image.tar: Dockerfile Cargo.toml $(wildcard src/*)
```

Comment:
This should mention the binary now instead of the source code I believe.

Reply:
yup, changed to the `target` dir as we now compile it to `target` being in `./test-plans`